src/content/doc-surrealdb/reference-guide/observability.mdx

```diff
@@ -0,0 +1,115 @@
+---
+sidebar_position: 6
```

Comment:
To keep the section alphabetical, this can be 4 and the other two following can be set to 5 and 6.

Reply:
Done